openshift/patches/override-rekt-images.patch

```diff
@@ -20,7 +20,7 @@ index 207c3d68..63858bce 100644
    containers:
      - name: flaker
 -      image: ko://knative.dev/eventing/test/test_images/event-flaker
-+      image: registry.ci.openshift.org/openshift/knative-v1.2.3:knative-eventing-test-event-flaker
++      image: registry.ci.openshift.org/openshift/knative-v1.2.1:knative-eventing-test-event-flaker
```

Comment:
promotion version is 1.2.3

Reply:
https://github.com/openshift/release/blob/8ad723db0fd5ba76a2f685b0fc1bdbdfd2ed9e84/ci-operator/config/openshift-knative/eventing-kafka-broker/openshift-knative-eventing-kafka-broker-release-v1.2__410.yaml#L171